css/css-multicol/resize-in-strict-containment-nested.html

```diff
@@ -4,7 +4,7 @@
 <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
 <div style="columns:2; column-gap:0; column-fill:auto; width:200px; height:100px; margin-left:-100px;">
-  <div style="columns:1;">
+  <div style="columns:1; column-gap:0;">
```

Comment:
Only one column (per outer fragmentainer) is expected, so specifying column-gap shouldn't make any difference. In fact, leaving it as non-zero (being explicit about it could be an option, though) is better.

Where is this being rendered incorrectly? I tested Chrome, Firefox, Safari. All green.